web/src/gateways/Assertion.gateway.ts

```diff
@@ -0,0 +1,18 @@
+import {endpoints} from '../redux/apis/Test.api';
+import {IAssertion} from '../types/Assertion.types';
+
+const {createAssertion, getAssertions, updateAssertion} = endpoints;
+
+const AssertionGateway = () => ({
```

Comment:
Facade to facilitate the usage of the async thunk functions from the redux API slice